src/dodal/devices/robot.py

```diff
@@ -23,7 +22,7 @@ def __init__(
             pv (str): The waveform PV that contains a list of strings
             index (int, optional): The index to read. Defaults to 0.
         """
-        self.bare_signal = epics_signal_r(Sequence[str_], pv)
+        self.bare_signal = epics_signal_r(Sequence[str], pv)
```

Comment:
Could: Does this mean we no longer need the cast to `str` below?

Reply:
We still need the cast. Not sure exactly where it happens, but reading the PV  converts this from a `str` to a `str_`